source/draw/RasterPacked.ooc

```diff
@@ -32,16 +32,6 @@ RasterPacked: abstract class extends RasterImage {
 		stride := this bytesPerPixel * size x
 		this init(ByteBuffer new(stride * size y), size, stride)
 	}
-	init: func ~fromOriginal (original: This) {
-		super(original size)
-		this _buffer = original buffer copy()
-		this _stride = original stride
-	}
-	init: func ~fromRasterImage (original: RasterImage) {
-		super(original size)
-		this _stride = this bytesPerPixel * original width
-		this _buffer = ByteBuffer new(this stride * original height)
-	}
```

Comment:
These two constructors had to be removed because they accept the same input types and the choice between them is decided at runtime depending on if original is packed or planar. `~fromOriginal` made a copy of the image while preserving padding while `~fromRasterImage` made a new image from the size without preserving padding.